apis.py

```diff
@@ -850,6 +850,36 @@ def task_completed(*args, **kwargs):
     return json.dumps({'Response': {}, 'status': success, 'string_response': response})
 
 
+@app.route('/admin-update-request', methods=['POST'])
+@login_required
+def task_completed(*args, **kwargs):
+    volunteer_id = kwargs['volunteer_id']
```

Comment:
volunteer_id has to come from front-end not from kwargs.
'user_id' of admin user comes from kwargs in case of @login_required

Reply:
Resolved.